kie-server-parent/kie-server-tests/kie-server-integ-tests-pmml/pom.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.kie.server</groupId>
+    <artifactId>kie-server-tests</artifactId>
+    <version>7.49.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kie-server-integ-tests-pmml</artifactId>
+
+  <name>KIE :: Execution Server :: Tests :: PMML-Trusty Integration Tests</name>
+  <description>KIE Execution Server Integration Tests (REST, JMS) with configuration options to run on different containers.</description>
+
+  <properties>
+    <java.module.name>org.kie.server.itests.pmml</java.module.name>
+    <org.drools.server.ext.disabled>false</org.drools.server.ext.disabled>
+    <org.kie.prometheus.server.ext.disabled>true</org.kie.prometheus.server.ext.disabled>
+    <org.kie.scenariosimulation.server.ext.disabled>true</org.kie.scenariosimulation.server.ext.disabled>
```

Comment:
Same as above

Reply:
those are "disabled".
I'll keep "disable" semantic all over.

Reply:
Whenever there is a referenced variable I'll leave it as-is.